no-littering.el

```diff
@@ -77,36 +77,64 @@ This variable has to be set before `no-littering' is loaded.")
   "The directory where packages place their persistent data files.
 This variable has to be set before `no-littering' is loaded.")
 
-(cl-flet ((etc (file) (expand-file-name (convert-standard-filename file)
-                                        no-littering-etc-directory))
-          (var (file) (expand-file-name (convert-standard-filename file)
-                                        no-littering-var-directory)))
-  (with-no-warnings ; many of these variables haven't been defined yet
-
-;;; Built-in packages
```

Comment:
I would like to keep the built-in and third-party packages separate and keep those headings. The built-in section should begin with a `no-littering-setq` form whose PACKAGE is `nil` and whose entries should be sorted alphabetically. After that come the package-specific `no-littering-setq`s, sorted alphabetically by PACKAGE.